src/execution/operator/csv_scanner/csv_state_machine.cpp

```diff
@@ -22,6 +22,12 @@ void CSVStateMachine::Reset() {
 	csv_buffer_iterator.Reset();
 }
 
+void CSVStateMachine::Transition(char current_char) {
```

Comment:
This should probably be inline in the header to avoid any potential performance impact